l10n_it_fatturapa_in/wizard/link_to_existing_invoice.py

```diff
@@ -88,7 +87,12 @@ class WizardLinkToInvoice(models.TransientModel):
 
     @api.model
     def _get_default_lines_vals(self, attachment):
-        fatt = get_invoice_obj(attachment)
+        fatt = attachment.get_invoice_obj()
+        if not fatt:
```

Comment:
Farei `if attachment.e_invoice_parsing_error` per maggiore chiarezza

Reply:
A me in realtà sembra già chiaro così, ma forse solo perché so come funziona `get_invoice_obj`: ho aggiunto la documentazione al metodo così risulta più chiaro anche ai posteri :smile:.

Eviterei di controllare la condizione `attachment.e_invoice_parsing_error` perché lo vedo come un _side effect_ del metodo il cui compito principale è ritornare `False` quando il parsing fallisce.
Inoltre eviterei di chiedere a Odoo il valore `e_invoice_parsing_error` quando non ci serve e possiamo controllare la stessa condizione con `fatt`, perché potrebbe venirgli in mente di recuperarlo dalla cache o ricalcolarlo.